cmd/opm/alpha/diff/cmd.go

```diff
@@ -44,27 +57,28 @@ func NewCmd() *cobra.Command {
 		Use:   "diff [old-refs]... new-refs...",
 		Short: "Diff old and new catalog references into a declarative config",
 		Long: templates.LongDesc(`
-Diff a set of old and new catalog references ("refs") to produce a
-declarative config containing only packages channels, and versions not present
-in the old set, and versions that differ between the old and new sets. This is known as "latest" mode.
-
-These references are passed through 'opm render' to produce a single declarative config.
-Bundle image refs are not supported directly; a valid "olm.package" declarative config object
-referring to the bundle's package must exist in all input refs.
-
-This command has special behavior when old-refs are omitted, called "heads-only" mode:
-instead of the output being that of 'opm render refs...'
-(which would be the case given the preceding behavior description),
-only the channel heads of all channels in all packages are included in the output,
-and dependencies. Dependencies are assumed to be provided by either an old ref,
-in which case they are not included in the diff, or a new ref, in which
-case they are included. Dependencies provided by some catalog unknown to
-'opm alpha diff' will not cause the command to error, but an error will occur
-if that catalog is not serving these dependencies at runtime.
-Dependency inclusion can be turned off with --no-deps, although this is not recommended
+'diff' returns a declarative config containing packages, channels, and versions
+from new-refs, optionally removing those in old-refs or those omitted by an include config file.
+
+Each set of refs is passed to 'opm render <refs>' to produce a single, normalized delcarative config.
+
+Depending on what arguments are provided to the command, a particular "mode" is invoked to produce a diff:
+
+- If in heads-only mode (old-refs is not specified), then the heads of channels in new-refs are added to the output.
+- If in latest mode (old-refs is specified), a diff between old-refs and new-refs is added to the output.
+- If --include-file is set, items from that file will be added to the diff:
+	- If --include-additive is false (the default), a diff will be generated only on those objects, depending on the mode.
+	- If --include-additive is true, the diff will contain included objects, plus those added by the mode's invocation.
```

Comment:
is it possible to include some optional help text with example calls?

Reply:
Done.